### DISCUSSION TITLE
Project Zomboid

### DISCUSSION THREAD

**Discussion by mvaessen** [gameserverapp/Platform#39] [Game request]
https://pzwiki.net/wiki/Dedicated_Server

**Comment by Decollates** [comment]
Bumping Project Zomboid. Since its resurgence and the .41+ updates, the game has had massive success and popularity and has almost matched 7 Days to Die’s popularity in terms of player count. It’s expected to get even crazier when it releases its update for animal husbandry and NPCs. PZ also has a very active modding community.

**Comment by JackRavenwood** [comment]
Oh 100% behind this - yes please. The next update is going to be HUGE

**Comment by gnarlyvr** [comment]
Would love to see PZ on GSA

**Comment by Eurymachus** [comment]
Would be amazing to have this ready for B42 \o/
